playbook_cloudkit_create_hosted_cluster.yml

```diff
@@ -14,11 +16,25 @@
     - role: cluster_settings
       vars:
         cluster_settings_template_id: "{{ cluster_order.spec.templateID }}"
-        cluster_settings_template_parameters: "{{ cluster_order.spec.templateParameters }}"
+        cluster_settings_template_parameters: "{{ cluster_order.spec.templateParameters | from_json }}"
 
     - role: hosted_cluster
       vars:
         hosted_cluster_name: "{{ cluster_order.metadata.name }}"
         hosted_cluster_namespace: "{{ cluster_working_namespace_result }}"
         hosted_cluster_settings: "{{ cluster_settings_result }}"
         hosted_cluster_state: present
+
+    - role: infrastructure/api
```

Comment:
generally, `ansible-lint` doesn't really like nested roles. Maybe the solution would be to split the roles in collections, e.g.: 
- `collections/cloudkit/hosted_cluster/roles/...`
- `collection/cloudlkit/infrastructure/roles/...`

Last time I tried it failed and I did not look for a solution, but I could be a better way to organize our stuff

Reply:
I have configured ansible-lint to disable the `role-name[path]` rule for now. I've updated everything to either fix the linting errors or to add `noqa:...` comments where I disagree with `ansible-lint`.

Reply:
Sure, let's merge this and then we can work on restructuring it.

Reply:
Created https://github.com/innabox/cloudkit-aap/issues/18 to track it